Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -914,6 +914,9 @@ - (void)presentThread:(TSThread *)thread
                 callOnViewAppearing:callOnViewAppearing];
         self.lastThread = thread;
 
+        // Never push more than one conversation view controller at a time.
```

Comment:
Wouldn't this logic be better in `pushTopLevelViewController:`?

In fact, it looks like it already is:
https://github.com/signalapp/Signal-iOS/blob/master/Signal/src/ViewControllers/HomeView/HomeViewController.m#L980

Is it not working? Can we fix that instead of adding more code?

Reply:
You're right, this part of the change looks redundant. Sorry, I don't have a repro so I can't test this.